testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -12376,6 +12376,9 @@ void testTimestampDiff(boolean coercionEnabled) {
     f.checkScalar("FORMAT_DATE('The date is: %x', DATE '2008-12-25')",
         "The date is: 12/25/08",
         "VARCHAR(2000) NOT NULL");
+    f.checkNull("FORMAT_DATE('%x', CAST(NULL AS DATE))");
```

Comment:
If there aren't tests where the format itself is null we should add some of those as well while we're at it/

Reply:
Hello, thank you for your review
```java
f.checkNull("FORMAT_DATE(NULL, DATE '2008-12-25'");
```
This way of writing shows parser error. Do I need to add comments?

Reply:
It depends what the anticipated behavior is. If a parser error is expected then that makes sense. If we expect a `null` result then something is wrong. Do you know what the expected behavior should be if a null format is passed?

Reply:
I ran a simple test against my BQ database (`SELECT FORMAT_DATE(null, CURRENT_DATE())`) and it returned `null` so I would expect this to return`null` as well unless other dialects behave differently.